client/layout/transient-notices/snackbar/index.js

```diff
@@ -92,6 +115,9 @@ function Snackbar(
 							isTertiary
 							onClick={ ( event ) => {
 								event.stopPropagation();
+								if ( explicitDismiss ) {
+									onRemove();
+								}
 								if ( onClick ) {
 									onClick( event );
 								}
```

Comment:
It would be nice to move this onClick handler to a non-anonymous function.

Reply:
Done in 52182f66333c9cebaaef1e1f9e1aced9b601ade2